.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "web"]
 	path = web
-	url = https://github.com/OpenPoGo/OpenPoGoWeb.git
```

Comment:
@solderzzc noted, please leave any suggestion here. 

I'll create another pull request with more clean commits and close this one.